lib/hutch/config.rb

```diff
@@ -13,6 +13,7 @@ def self.initialize(params={})
         mq_host: 'localhost',
         mq_port: 5672,
         mq_exchange: 'hutch',  # TODO: should this be required?
+        mq_exchange_options: {},
```

Comment:
Since this is a new functionality I think we can put a warning comment near this option. If somebody looks to find which is the required configuration key for this, will see that default is durable and also will see the comment.